style.css

```diff
@@ -0,0 +1,816 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+html {
+  overflow-x: hidden;
+}
+
+ul {
+  list-style: none;
+}
+
+body {
+  background-color: #f4f5f6;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* desktop navbar */
+
+.top_nav {
+  background-color: #f72210;
+  height: 95px;
+}
+
+img:hover {
+  cursor: pointer;
+}
+
+.top_nav > div > img {
+  width: 366px;
+  height: 85px;
+  cursor: pointer;
```

Comment:
Create a new class for `cursor: pointer;`. Then add this class to all the elements that need it in the HTML file. Let's keep the DRY rule respected.